src/httpclient.js

```diff
@@ -0,0 +1,282 @@
+/*
```

Comment:
Can we document why this is included? I assume it has to do with not getting updates to scoped-http-client.

Reply:
Yes. Do you have a suggestion where to document that? Yes, the issue was scoped-http-client wasn't maintained anymore.

Reply:
I was thinking a comment at the top of the file. Would be good to move off this, but will be a pain because of how much plugins depend on it 😒